great_expectations/data_context/store/store_backend.py

```diff
@@ -391,7 +400,10 @@ def _set(self, key, value, content_encoding='utf-8', content_type='application/j
         import boto3
         s3 = boto3.resource('s3', **self._boto3_options)
         result_s3 = s3.Object(self.bucket, s3_object_key)
-        result_s3.put(Body=value.encode(content_encoding), ContentEncoding=content_encoding, ContentType=content_type)
+        if type(value) is bytes:
+            result_s3.put(Body=value)
```

Comment:
I think we still want to set content_type / content_encoding here

Reply:
K. Added logic to get content_type and content_encoding using mimetype.guess_type(), which is extension based. Tested it out with files that are currently in `static`, and get content_types just fine, but guess_type always yields None for encoding. Unfamiliar with content_encoding when it comes to non-text files. Since we're getting bytes using f.read() in binary mode, would encoding just be UTF-8?